R/format_PEW_temp.R

```diff
@@ -0,0 +1,541 @@
+                  FemaleID, MaleID,
+                  ClutchType_observed, ClutchType_calculated,
+                  LayDate_observed, LayDate_min, LayDate_max,
+                  ClutchSize_observed, ClutchSize_min, ClutchSize_max,
+                  HatchDate_observed, HatchDate_min, HatchDate_max,
+                  BroodSize_observed, BroodSize_min, BroodSize_max,
+                  FledgeDate_observed, FledgeDate_min, FledgeDate_max,
+                  NumberFledged_observed, NumberFledged_min, NumberFledged_max,
+                  AvgEggMass, NumberEggs, AvgChickMass, NumberChicksMass,
+                  AvgTarsus, NumberChicksTarsus, OriginalTarsusMethod, ExperimentID)
+
+  return(Brood_data)
+
+}
+
+create_capture_PEW <- function(pew_data, Brood_data) {
+
+  Brood_data_sel <-
+    Brood_data %>%
+    dplyr::select(BreedingSeason, Species, PopID, BroodID, LocationID,
+                  HatchDate_observed)
+
+
+  Capture_data_temp <-
+    pew_data %>%
+    #### Rename variables
+    dplyr::rename(LocationID = NestboxID) %>%
+    dplyr::select(-DateRingingChicks) %>%
+    #### Create new variables
+    dplyr::mutate(#### ASSUMPTION: There is no captureID info, so I made this up
+                  CaptureID = paste(IndvID, LocationID,
+                                    paste0(lubridate::year(CaptureDate),
+                                           sprintf("%02d", lubridate::month(CaptureDate)),
+                                           sprintf("%02d", lubridate::day(CaptureDate))),
+                                    sep = "_"),
+      #### ASSUMPTION: Some times are strange, to simplify, round to whole hour
+      #### ? do they correspond only to the experimental observations? keep/remove?
+      CaptureTime = ifelse(is.na(ObservationTimeH),
+                           NA,
+                           paste(sprintf("%02d", round(ObservationTimeH, 0)),
+                                 "00", sep = ":")),
+      Sex_observed = ifelse(Sex == "Chick", NA_character_, substr(Sex, 1, 1)),
+      CaptureAlive = ifelse(Method != "Found dead", TRUE, FALSE),
+      #### ASSUMPTION: the same as CaptureAlive, as there is no additional data
+      ReleaseAlive = CaptureAlive,
+      CapturePopID = PopID,
+      CapturePlot = NA_character_,
+      ReleasePopID = ifelse(ReleaseAlive == TRUE, CapturePopID, NA_character_),
+      ReleasePlot = ifelse(ReleaseAlive == TRUE, CapturePlot, NA_character_),
+      #### ASSUMPTION: use NA, as there is no information
+      OriginalTarsusMethod = NA_character_,
+      WingLength = NA,
+      ExperimentID = dplyr::case_when(Experiment == "BSM - Griffioen et al. 2019 PeerJ" ~
+                                        "COHORT; PARENTAGE",
+                                      Experiment == "2h Temp D4" ~ "SURVIVAL",
+                                      Experiment == "2h Temp D4 + Handicaping Male: Griffioen et al. 2019 Front Ecol&Evol" ~
+                                        "SURVIVAL; PARENTAGE"),
```

Comment:
Brood ExperimentID and Capture ExperimentID don't necessarily need to be the same. For example, a cross foster experiment will affect the parentage of a brood and the chicks, but not of the parents.

Reply:
I am not quite sure what you mean. Is it that there should/could be different ExperimentID in the Capture_data and in the Brood_data?

Reply:
Exactly. Some experiments will affect the brood and parents differently. For example, supplemental feeding of adults might improve adult survival but only change brood phenology.